packages/client/src/Components/Mapping/components/Adjudication/index.jsx

```diff
@@ -29,17 +30,25 @@ const Adjudication = ({ store, onStoreUpdate, callbackFunction }) => {
 
   newStore.formType = 'adjudication'
 
-  // HACK: This is just for the current wargame, this will need to be replaced with a non text-based comparison
+  const conditions = allPlatforms[0].conditions
```

Comment:
This is a mistake - it's using the conditions available for the first platform listed.  We have to use the conditions for the `platformType` of this asset.

Reply:
On it.

Reply:
Done. Sorry about that, that's the peril of writing debug code and not flagging it!